### DISCUSSION TITLE
calendarTodayLabel in datetime

### DISCUSSION THREAD

**Discussion by saiichihashimoto** [sanity-io/sanity#3354] [Sanity Studio Support]
For the datetime type, these options are laid out:

<img width="726" alt="image" src="https://user-images.githubusercontent.com/2819256/174111844-dbfdebf1-29bd-4d0e-bdd2-dccaea9dbb6c.png">

In the examples, with all options sets, these options are laid out:

<img width="722" alt="image" src="https://user-images.githubusercontent.com/2819256/174111937-8f98a949-1b8a-455b-b9ce-49591f8d3529.png">

Again, in `@sanity/types`, these options are laid out:

<img width="262" alt="image" src="https://user-images.githubusercontent.com/2819256/174112001-bfb37373-3e39-4ba1-ba08-23f46c2c2cce.png">

Is `calendarTodayLabel` a real option?

**Comment by p10e** [comment]
Good catch, @saiichihashimoto, thanks! It seems like `calendarTodayLabel?: string` is missing from `@sanity/types` indeed. The option itself should be available as per [DateInput.tsx](https://github.com/sanity-io/sanity/blob/v3/packages/sanity/src/form/inputs/DateInputs/DateInput.tsx#L15) and [DateTimeInput.tsx](https://github.com/sanity-io/sanity/blob/v3/packages/sanity/src/form/inputs/DateInputs/DateTimeInput.tsx#L22) on the v3 branch.

**Reply by saiichihashimoto** [reply]
I added it to the @sanity/types in a PR #3355

**Reply by p10e** [reply]
Much appreciated, @saiichihashimoto! We'll make sure to get eyes on this.
